### PR TITLE
General HTML improvements

### PR DESCRIPTION
**Pull request by hsoj95**
Added links to the GitHub page for the project for the convenience of
the user.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
     <p>In addition to matching you to the eight values, the quiz also attempts to match you to a political ideology. This is a work in progress and is much less accurate than the values and axes, so don't take it too seriously. If you disagree with your assigned ideology, send us an email at eightvalues@gmail.com with your scores, matched ideology, and preferred ideology, and we'll look into adjusting the system. Thanks!</p>
 <h2>I don't like my scores!</h2>
     <p>¯\_(ツ)_/¯<br/>
-    If you have any suggestions or constructive criticism, feel free to send it to eightvalues@gmail.com or open an issue on the GitHub page!</p>
+    If you have any suggestions or constructive criticism, feel free to send it to eightvalues@gmail.com or open an issue on the GitHub page here: <a href="https://github.com/8values/8values.github.io">8values.GitHub</a></p>
```
Comment (Saphareas) [inline index.html]: Could you also add a link for the eMail?
`<a href="mailto:eightvalues@gmail.com">eightvalues@gmail.com</a>`
also I would simply do it like this:
_… issue on the_ `<a href="https://github.com/8values/8values.github.io">GitHub page</a> `

**Comment by Saphareas** [comment]
@elifoster Well, I thought this was clear. ^^

**Comment by hsoj95** [comment]
Yes, I'll be happy to do both! With the link name, I honestly wasn't sure how it should be displayed, so I just took an educated guess at what would look good. So I'm happy for the suggestion on that! As for the mail link, I actually considered doing this to begin with, but I had heard before from others, not associated with this project, that it wasn't always a good idea to have a hyper-link to a mail address, for spam reasons. I don't know if this is a legitimate concern or not, but I'll be more than happy to add the link to the mailing address as well.

**Comment by hsoj95** [comment]
Ok, link names have been updated and hyperlinks have been added to the email addresses! Let me know if there's anything else you think should be changed! 🙂

**Comment by hsoj95** [comment]
So I went back and was looking at the links I added and I had a question. The links are the standard default blue color that's underlined, since the anchor tags only inherit the styling from the page.  My question would be, is the default color and underline good, or should it be different? If y'all think it should be different, what color and style should they be? I'm honestly ok with the blue color, but i figured I'd ask just in case! Any suggestions are welcome!

**Comment by hsoj95** [comment]
I decided to add two HTML comments, one to the quiz and the other to the results to show which specific script tag was used for the JavaScript that the test actually uses. It makes it a lot more clear as it was rather confusing with just the one comment about web analytics.

**Comment by hsoj95** [comment]
Changed the name of the Pull Request because it has now branched out beyond just adding HTML links to the pages.

**Comment by hsoj95** [comment]
As was suggested in an issue request, #46, I have added a line dynamically showing the number of questions currently in the survey.

**Comment by hsoj95** [comment]
This pull request is complete on my end! It's just ready to be merged if and/or when y'all think it is ready! 🙂

**Comment by hsoj95** [comment]
Is development on this project still occurring? Just wondering due to the length of time this pull request, and the other pull request as well, have been open with no response.

**Comment by TristanBomb** [comment]
Unfortunately, I have neither the time nor interest to continue this project myself. However, there does seem to be a number of people still interested in this project. I have merged the PR, but if anyone is interested in being an official developer, please contact me or reply to [this thread](https://github.com/8values/8values.github.io/issues/62). Thanks!

**Pull request merged**